README.md

```
@@ -103,26 +103,21 @@ For instance:
 
 ```
 docker build --tag myeventstore . \
---build-arg CONTAINER_RUNTIME=bookworm-slim \
+--build-arg CONTAINER_RUNTIME=noble \
 --build-arg RUNTIME=linux-x64
 ```
 
 **_Note:_** Because of the [Docker issue](https://github.com/moby/buildkit/issues/1900), if you're building a Docker image on Windows, you may need to set the `DOCKER_BUILDKIT=0` environment variable. For instance, running in PowerShell:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)